src/cljs/velho_ds/pages/main.cljs

```diff
@@ -100,7 +97,34 @@
      [fields/dropdown-menu {:heading "Text"
                             :selected-fn #(js/alert (str "Selected value: " %))
                             :options values
-                            :no-selection-text "- No selection -"}])])
+                            :no-selection-text "- No selection -"}])
+   [:h2 "Grid"]
+   [grid/grid-wrap {:rows 3
+                    :cols 3}
+    [grid/grid-cell {:col-start 1
+                     :col-end 4
+                     :style {:background-color "springgreen"
+                             :text-align "center"
+                             :border "1px solid silver"}} [:p "test"]]
+    [:div {:style {:background-color "whitesmoke"
```

Comment:
Normaali elementit gridin sisällä menee normaalisti vasemmalta oikeella ja rivittyy.

Reply:
I'd change the "springgreen" background to something a bit less nineties. :)